contentcuration/contentcuration/tests/base.py

```diff
@@ -122,34 +123,33 @@ def setUp(self):
         self.channel = testdata.channel()
         self.user = testdata.user()
         self.channel.editors.add(self.user)
-        token, _new = Token.objects.get_or_create(user=self.user)
-        self.header = {"Authorization": "Token {0}".format(token)}
+        self.token, _new = Token.objects.get_or_create(user=self.user)
         self.client = APIClient()
-        self.client.force_authenticate(self.user)
+        self.client.force_authenticate(self.user)   # This will skip all authentication checks
```

Comment:
Is it possible to log out of this forced authentication? Might be safer to just log in as the user, rather than use the force authentication?

Reply:
Yes. When writing authentication related tests --- anything that exerices the sessionauth or tokeauth decorators, you can "force logout" the APIClient by calling this method with `user=None` like here: 
https://github.com/learningequality/studio/pull/1453/files#diff-ca452a7728af35ddf95fa812d1c69347R53

Reply:
Thanks - shortly after writing this, I discovered it to be true in my own tests, so apologies for extracting the unnecessary explanation!